src/main/webapp/app/lecture/lecture-attachments.component.ts

```diff
@@ -130,6 +131,9 @@ export class LectureAttachmentsComponent implements OnInit, OnDestroy {
             this.attachmentService.create(this.attachmentToBeCreated!, this.attachmentFile!).subscribe({
                 next: (attachmentRes: HttpResponse<Attachment>) => {
                     this.attachments.push(attachmentRes.body!);
+                    this.lectureService.findWithDetails(this.lecture.id!).subscribe((lectureResponse: HttpResponse<Lecture>) => {
+                        this.lecture = lectureResponse.body!;
+                    });
```

Comment:
It is almost like that everywhere in the code, should there be an issue to change them as well? Because I was referring to them and it looks a bit confusing

Reply:
I will have a discussion with @JohannesWt, who is more profound on this topic, there are some particular cases where Angular manages to unsubscribe automatically

I will let you know - feel free to leave it as it is then and in case we need to change it we will do it in a separate issue 👍